garden/tutorials/building_robot/building_robot.sdf

```diff
@@ -6,16 +6,16 @@
             <real_time_factor>1.0</real_time_factor>
         </physics>
         <plugin
-            filename="ignition-gazebo-physics-system"
-            name="ignition::gazebo::systems::Physics">
+            filename="libgz-sim-physics-system.so"
+            name="gz::sim::systems::Physics">
         </plugin>
         <plugin
-            filename="ignition-gazebo-user-commands-system"
-            name="ignition::gazebo::systems::UserCommands">
+            filename="libgz-sim-user-commands-system.so"
```

Comment:
```suggestion
            filename="gz-sim-user-commands-system"
```

Reply:
There are a number of places in this PR that require this change.

Reply:
Well, in that case, markdown files need this changes too. I can do that.